scenarios/zero_arc_create_and_read/README.md

```diff
@@ -0,0 +1,50 @@
+## zero_arc_create_and_read
+
+### Description
+
+A zero-arc/full-arc mixed scenario with two types of zero arc nodes, ones that create data and ones that read data, as well as full arc nodes to "relay" the data. The scenario has three roles:
+
+- _zero_write_: A zero arc conductor that just creates entries with a timestamp field. Those entries are linked to a known base hash so that full arc nodes can retrieve them.
```

Comment:
```suggestion
- _zero_write_: A zero arc conductor that just creates entries with a timestamp field. Those entries are linked to a known base hash so that _zero_read_ nodes can retrieve them.
```

Reply:
addressed :+1: